.github/workflows/apcd-cms.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [main]
     paths:
       - "apcd_cms/**"
```

Comment:
```
  paths:
      - "apcd_cms/**"
  ```

this means both `sync` and `build_commit` only run when there are changes in apcd_cms directory in main.  is that desired for `sync`?

Reply:
Good catch my mistake! I intended for all pushes to main to trigger the sync for our develop branch, not just apcd_cms.

Reply:
Why would you sync with what is happening in other CMS directories that are unrelated to apcd_cms? If you don't filter this way, you will get a merge triggered everry time someone updates an asset in any other project.